src/components/footer/footer.component.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { Link } from 'gatsby'
+
+import { useFooterStyles } from './footer.styles'
+// import { Container } from "../../shared/container/container.component"
+
+export const Footer = () => {
+  const classes = useFooterStyles()
+
+  return (
+    <footer className={classes.footer}>
+      {/* <Container> */}
```

Comment:
I would remove this `Container` references if they are not needed

Reply:
Yes. Done.